### PR TITLE
doc: fix document about rados mon

### PR DESCRIPTION
**Pull request by Liuchang0812**
Signed-off-by: liuchang0812 <liuchang0812@gmail.com>

### CODE REVIEW

**Comment by Liuchang0812** [comment]
@tchaikov hi, tchaikov, take a look please.  remove log message and correct format.

File: doc/rados/operations/control.rst [hunk 1]
```diff
@@ -321,89 +321,142 @@ Mon Subsystem
...
-	2011-12-14 10:43:08.015333 mon {- [mon,dump]
-	2011-12-14 10:43:08.015567 mon.0 -} 'dumped monmap epoch 1' (0)
-	epoch 1
-	fsid 444b489c-4f16-4b75-83f0-cb8097468898
-	last_changed 2011-12-12 13:28:27.505520
-	created 2011-12-12 13:28:27.505520
-	0: 127.0.0.1:6789/0 mon.a
-	1: 127.0.0.1:6790/0 mon.b
-	2: 127.0.0.1:6791/0 mon.c
+	epoch 2
```
Comment (tchaikov) [inline doc/rados/operations/control.rst]: missing this line before `epoch 2`:
```
dumped monmap epoch 2
```

Comment (Liuchang0812) [inline doc/rados/operations/control.rst]: yes...I'm sorry. fix it now

**Comment by Liuchang0812** [comment]
@tchaikov done :smile:

**Pull request merged**